From 3ed4020489805ad66e0612a036959e117087607d Mon Sep 17 00:00:00 2001
From: Noam.Lessner <nlessner@habana.ai>
Date: Fri, 05 Nov 2021 09:55:46 +0200
Subject: [PATCH] [SW-0] gaudi2: pcie: increase fw size in makefile

Adjusted Makefile size limit to fit new FW size of pcie phy.

Change-Id: I01660bf8f8b24832f40a527a43494c3236444e81
---

diff --git a/arch/gaudi2/arm/Makefile b/arch/gaudi2/arm/Makefile
index e2d6aa8..6778b88 100644
--- a/arch/gaudi2/arm/Makefile
+++ b/arch/gaudi2/arm/Makefile
@@ -1,6 +1,6 @@
 include $(HL_DEV_ARCH_COMMON)/Makefile
 
-FW_SIZE := 0x4400
+FW_SIZE := 0x5400
 PPBOOT_MAX_SIZE := 20480
 PREBOOT_MAX_SIZE := 64512
 
